samples/sql-bridge/Version_6/MsmqToSqlRelay/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Iesi.Collections" version="4.0.1.4000" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="8.0.2" targetFramework="net452" />
+  <package id="NHibernate" version="4.0.4.4000" targetFramework="net45" />
+  <package id="NServiceBus" version="6.0.0-unstable1461" targetFramework="net452" />
+  <package id="NServiceBus.NHibernate" version="6.3.0-pullrequest0153" targetFramework="net452" />
```

Comment:
@petersgiles this looks like it is done https://github.com/Particular/NServiceBus.NHibernate/pull/153  do u want to merge and deploy it?